src/actions/port.actions.js

```diff
@@ -44,8 +41,8 @@ export const setPortGraphicalAttribute = (portId, graphicalAttributes) => ({
  * @param {string} portId
  * @param {string} graphicalAttributesKey - the key of the attribute to be removed
  */
-export const removePortGraphicalAttribute = (portId, graphicalAttributesKey) => ({
-	type: FLOWDESIGNER_PORT_REMOVE_GRAPHICAL_ATTRIBUTES,
+export const removePortGraphicalAttribute = (portId: string, graphicalAttributesKey: string): PortAction => ({
+	type: 'FLOWDESIGNER_PORT_REMOVE_GRAPHICAL_ATTRIBUTES',
```

Comment:
You don't use constant anymore.
could you ensure the constant is also checked by flow type ?